README.md

```diff
@@ -67,6 +67,12 @@ Table of Contents:
 | **[Terraform NGINX hello world](containers/terraform-nginx-hello-world/README.md)** <br/> A minimal example running the base NGINX image in a serverless container deployed with Terraform. | N/A          | [Terraform]            |
 | **[Triggers with Terraform](containers/terraform-triggers/README.md)** <br/> Configuring two SQS triggers, used to trigger two containers, one public, one private.                         | N/A          | [Terraform]            |
 
+### Jobs
+
+| Example                                                                                                                                                                                     | Language     | Deployment             |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|------------------------|
+| **[Serverless MLOps](jobs/ml-ops/README.md)** <br/> An example to run a Serverless Machine Leaning workflow                                                                                 | Python       | [Terraform]+[Console]  |
```

Comment:
For now, I used console to define and run a job until Scw Terraform's module for jobs is released.

Reply:
Yeah no pb!